Sources/Player/Player.swift

```diff
@@ -360,6 +360,26 @@ public extension Player {
 		}
 	}
 
+	func offline(mediaProduct: MediaProduct) -> Bool {
```

Comment:
[suggestion] considering these are all public, maybe we can add some comments to each func.